container/service_test.go

```diff
@@ -36,25 +36,13 @@ func TestStartService(t *testing.T) {
 
 func TestStopService(t *testing.T) {
 	namespace := []string{"namespace"}
-	containerData := []types.Container{}
-	containerJSONData := types.ContainerJSON{
-		ContainerJSONBase: &types.ContainerJSONBase{
-			State: &types.ContainerState{},
-		},
-	}
-
 	dt := dockertest.New()
 	c, _ := New(ClientOption(dt.Client()))
 
-	dt.ProvideContainerList(containerData, nil)
-	dt.ProvideContainerInspect(containerJSONData, nil)
+	dt.ProvideServiceRemove(nil)
+	dt.ProvideServiceInspectWithRaw(swarm.Service{}, nil, dockertest.NotFoundErr{})
```

Comment:
can we test call arguments of _ServiceInspectWithRaw()_ as well, with _LastServiceInspectWithRaw()_ util.

Reply:
Done